readme.md

```diff
@@ -673,6 +676,8 @@ You can use any assertion library instead of or in addition to the built-in one,
 
 This won't give you as nice an experience as you'd get with the [built-in assertions](#assertions) though, and you won't be able to use the [assertion planning](#assertion-planning) ([see #25](https://github.com/avajs/ava/issues/25)).
 
+You'll have to configure AVA to not fail tests if no assertions are executed, because AVA can't tell if custom assertions pass. Set the `failWithoutAssertions` option to `false` in AVA's [`package.json` configuration](#configuration).
```

Comment:
This feature, while useful, is going to hit a lot of people. Any chance we could add support for assertions throwing `AssertionError`?

I know we have https://github.com/avajs/ava/issues/1094, but that's more about integrating with `t` and `t.plan()`. I sometimes use assertions helper, and would like to use them in AVA without having to do anything extra.